### PR TITLE
helm: add page

### PR DESCRIPTION
**Pull request by VidhiG**
This adds a page for helm commands.

[Helm](https://helm.sh/) is a package manager for Kubernetes, that helps in defining, installing and updating k8s applications.

### CODE REVIEW

File: pages/common/helm.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# helm
+
+> Helm is a package manager for Kubernetes, that helps in defining, installing and updating k8s applications.
```
Comment (agnivade) [inline pages/common/helm.md]: We can move the second part of the description as a sub-heading. It is too long for a single line.

**Comment by VidhiG** [comment]
@sbrl @agnivade 
I have made the updates that were requested and signed the CLA too. Let me know if I should make any other changes.

Thank you

**Comment by agnivade** [comment]
It looks like the email that you have used to push the commits and your github profile email are different. Can you squash all the commits and force push again using the email for `VidhiG` ? That should resolve the issue.

File: pages/common/helm.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# helm
+
+> [Helm](https://helm.sh/) is a package manager for Kubernetes.
```
Comment (agnivade) [inline pages/common/helm.md]: tldr is a command line tool. So markdown syntax like this does not work. Just say Helm

File: pages/common/helm.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- Delete helm repository:
+
+`helm repo remove {{repo_name}}`
+
+- Install helm chart:
+
+`helm install {{repo_name}}/{{chart_name}}`
+
+- Download helm chart tar:
```
Comment (agnivade) [inline pages/common/helm.md]: Probably "Download helm chart as a tar archive" sounds better

File: pages/common/helm.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+`helm get {{chart-release-name}}`
```
Comment (agnivade) [inline pages/common/helm.md]: chart_release_name

File: pages/common/helm.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# helm
+
+> [Helm](https://helm.sh/) is a package manager for Kubernetes.
+
+- Create helm chart:
```
Comment (sbrl) [inline pages/common/helm.md]: Perhaps `Create a helm chart` would read better?

File: pages/common/helm.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+`helm create {{chart_name}}`
+
+- Add helm repository:
```
Comment (sbrl) [inline pages/common/helm.md]: Perhaps `Add a new helm`.... would read better?

File: pages/common/helm.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+- List helm repositories:
+
+`helm repo list`
+
+- Update helm repositories:
+
+`helm repo update`
+
+- Delete helm repository:
```
Comment (sbrl) [inline pages/common/helm.md]: Perhaps `Delete a helm`.... would read better?

**Comment by VidhiG** [comment]
@sbrl @agnivade 
Made the requested changes.
Squashed the commits and pushed using the github email

**Pull request merged**